### ISSUE TITLE
Cannot re-create a interpipesink because of not unique

### ISSUE THREAD

**Issue by b04505009** [RidgeRun/gst-interpipe#85]
I would like to reconstruct my pipeline after it received EOS. But after I set it state to NULL, unref and reconstruct, previous interpipesink seems still left in hashtable, thus my interpipesrc cannot reconnect to the new interpipesink. How can I remove this record from hashtable if I don't want to include this library and call gst_inter_pipe_remove_node()?

**Comment by jsalas98** [comment]
Hi @b04505009 we are currently working in this issue. I created a dummy app to reproduce it and I was able to see that same error, but it would be great if you could share the app you are testing with.

**Comment by b04505009** [comment]
Hi @jsalas98 thank you for your help. Sorry I cannot share my application with you, and also my application may be too complicated to demonstrate this issue. But I can make a simpler version app to reproduce the issue if it can really help you a lot. Thanks again!

**Comment by jsalas98** [comment]
Sure @b04505009, that would work too. It is just to make sure that the issue I am seeing is happening under your same circumstances.

**Comment by b04505009** [comment]
@jsalas98 
Sorry for late reply. Due to some policy, our development is not using gstreamer, so I can support you on this issue only in my free time. I would try to produce the sample on this weekend. Could you wait until then? Thank you

**Comment by jsalas98** [comment]
@b04505009 Yeah, that's okay, I can wait. Just in case, in the dummy app I created that error was related to how I was reading EOS messages with the bus and then recreating the pipeline in that same bus callback.
